.tekton/central-db-pull-request.yaml

```diff
@@ -221,12 +220,43 @@ spec:
       - name: basic-auth
         workspace: git-auth
 
+    - name: determine-image-tag
```

Comment:
Sorry, once again regarding the placement of this task in the pipeline. Yann's PR
https://github.com/stackrox/stackrox/pull/10851 made me look again at the [init task](https://github.com/redhat-appstudio/build-definitions/blob/main/task/init/0.2/init.yaml) and I realized that `init` should go _after_ we determine the tag.
For us right now it does not matter since we want to always rebuild and the wrong tag that `init` checks does not play the role, but it may matter in some future when colleagues add more responsibilities to `init` and/or when we contribute `determine-image-tag`.

Again, no ready solution for that on my side. Do you have any thoughts how to best approach it?

Reply:
> in some future when colleagues add something to `clone-repository` that requires `init` to have been called (eg. placing some contents in the workspace source before the checkout)

To me this feels made up because `init` does not attach any workspace, but nevertheless I share the uncertainty about `init` because it is ambiguously described and to me it isn't clear _what `init` will **not** be_ responsible for.

It's own description
> Initialize Pipeline Task, include flags for rebuild and auth. Generates image repository secret used by the PipelineRun.

does not help, and besides the image repository secret part is a lie according to [its own source](https://github.com/redhat-appstudio/build-definitions/blob/82f7008ec8ecbfd8f4c1218fff3924d97730eb71/task/init/0.2/init.yaml#L37-L48).

I don't feel we're standing on a solid ground with the Konflux team sharing our vision. Therefore, I think the safest is to align on our use with the Konflux team, as we discussed in [today's sync](https://docs.google.com/document/d/1bCKSof0166b5M0e9hvgMBgAPzU9-jH-dEHDTjqrHd54/edit#heading=h.m9hruc80pomn).

Reply:
FTR https://redhat-internal.slack.com/archives/C05TS9N0S7L/p1715096492694009?thread_ts=1714998487.394219&cid=C05TS9N0S7L